pkgs/development/tools/git-ps-rs/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, libgpg-error
+, gpgme
+, dbus
+, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "git-ps-rs";
+  version = "6.5.0";
+
+  nativeBuildInputs = [ pkg-config libgpg-error gpgme dbus ];
```

Comment:
Only pkg-config and gpgme cannot be moved.

Reply:
I'm not certain that it is being linked against. It seems just a gpg command is being run during configuration of one of the cargo dependencies.

Reply:
You are correct that for the build of `git-ps-rs` it is not needed, but this is what happens when I move it to buildInputs:
```
nix-repl> :b git-ps-rs
error: builder for '/nix/store/15vk6cgw6n9565hgvxf31rbx71ic6jfd-git-ps-rs-6.5.0.drv' failed with exit code 101;
       last 10 log lines:
       >   cargo:rerun-if-env-changed=LIBGPG_ERROR_LIB_DIR
       >   cargo:rerun-if-env-changed=LIBGPG_ERROR_LIBS
       >   cargo:rerun-if-env-changed=LIBGPG_ERROR_CONFIG
       >
       >   --- stderr
       >   running: "sh" "-c" "gpg-error-config --version"
       >   sh: line 1: gpg-error-config: command not found
       >   command did not execute successfully, got: exit status: 127
       >   Error: ()
       > warning: build failed, waiting for other jobs to finish...
       For full logs, run 'nix log /nix/store/15vk6cgw6n9565hgvxf31rbx71ic6jfd-git-ps-rs-6.5.0.drv'.
```

Reply:
In fact, I had my own flake for this project that I was using to build it and I only included gpgme into buildInputs.

Reply:
Didn't know that gpgme contains a small script. Can you add a very short comment next to it, so that people know this?

pkg-config is not a lib, it always goes to nativeBuildInputs